src/components/WorkflowCanvas.tsx

```
@@ -1209,6 +1322,22 @@ export function WorkflowCanvas() {
+              case "imageCompare":
+                return "#14b8a6"; // teal-500 - for comparison
+              case "loop":
+                return "#a855f7"; // purple-500 - for loops/iteration
+              case "maskInpaint":
+                return "#d946ef";
+              case "colorPalette":
+                return "#ec4899"; // pink-500 - for color palette
+              case "imageFilter":
+                return "#14b8a6";
+              case "loop":
+                return "#a855f7"; // purple-500 - for loops/iteration
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)